sv_inline.h

```diff
@@ -903,7 +903,7 @@ Perl_SvPV_helper(pTHX_
             return SvPVX_mutable(sv);
         }
         else if(return_flags & SV_CONST_RETURN) {
-            return (char *) SvPVX_const(sv);
+            return (char *)(size_t)SvPVX_const(sv);
```

Comment:
Can you explain this cast? `SvPVX_const` returns a `SvPVX_const`, why would you cast via a `size_t`?